### PR TITLE
LPS-136359 Add Transitions to WorkflowDefinition model entity and expose this information on Headless’s Workflow API

### PR DESCRIPTION
**Pull request by MarinhoFeliphe**
Story: [LPS-133990](https://issues.liferay.com/browse/LPS-133990)

### CODE REVIEW

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:stop

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

File: modules/apps/portal-workflow/portal-workflow-web/src/test/java/com/liferay/portal/workflow/web/internal/util/filter/WorkflowDefinitionImpl.java [hunk 1]
```diff
@@ -100,6 +102,11 @@ public int getVersion() {
 		return 0;
 	}
 
+	@Override
+	public List<WorkflowTransition> getWorkflowTransitions() {
+		return null;
+	}
+
```
Comment (rafaprax) [inline modules/apps/portal-workflow/portal-workflow-web/src/test/java/com/liferay/portal/workflow/web/internal/util/filter/WorkflowDefinitionImpl.java:109]: @MarinhoFeliphe this change should be part of this commit https://github.com/liferay-workflow/liferay-portal/pull/466/commits/7d47b20d4fd5ada6e258e768bd32022e090880df instead.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:forward

**Pull request closed without merging**